app/controllers/users_controller.rb

```diff
@@ -1,4 +1,35 @@
 class UsersController < ApplicationController
+  def show
+    id = params[:id]
+    if(User.find_by_id(id).nil?)
+      flash.now[:alert] = "User not found"
+      render "static_pages/home"
+    else
+      @user = User.find(id)
+    end
+    # debugger
+  end
+  
   def new
+    @user = User.new
+  end
+  def create
+    @user = User.new user_params
+    if @user.save
```

Comment:
```suggestion
    if @user.save!
```

save! khác với save như thế nào?